src/main/resources/templates/fragments/head.html

```diff
@@ -20,8 +20,12 @@
 
     <link href="../css/sidemenu.css" rel="stylesheet" />
 
+   
+
     <link rel="stylesheet" href="../js/codemirror/lib/codemirror.css" />
 
+    <link rel="stylesheet" href="../js/Monaco-Editor/dev/vs/editor/editor.main.css" />
+
     <link rel="stylesheet" href="../js/codemirror/theme/xq-dark.css">
 
```

Comment:
When I make a code change, I see that the editor on sidepanel throws a error and I am NOT able to SAVE my changes.
<img width="1481" alt="image" src="https://github.com/ESIPFed/Geoweaver/assets/45752727/9e4d3b7f-7ac1-4878-99d7-7abc5147d54b">

Here is another error that I encountered
<img width="1503" alt="image" src="https://github.com/ESIPFed/Geoweaver/assets/45752727/23fb1245-47ac-44f6-b2d6-9541c21b4941">


1. The history tab does not work and is stuck on loading for the sidepanel
2. On bigger screen the height is uneven compared to logging

<img width="1509" alt="image" src="https://github.com/ESIPFed/Geoweaver/assets/45752727/87dfc9fc-3ebc-47df-83af-cb1cd85b19f3">

Reply:
will look into it,
the history tab again uses an instance of codeMirror, will have to make changes to that as well.